src/lib/sender.js

```diff
@@ -189,6 +196,40 @@ function createSender (opts) {
   }
 
   /**
+   * Create a payment request using an SSP shared secret.
+   *
+   * @param {Object} params Parameters for creating payment request
+   * @param {String} params.destination_amount Amount that should arrive in the recipient's account
+   * @param {String} params.destination_account Target account's ILP address
+   * @param {String} [params.id=uuid.v4()] Unique ID for the request (used to ensure conditions are unique per request)
+   * @param {String} [params.expires_at=30 seconds from now] Expiry of request
+   * @param {Object} [params.data=null] Additional data to include in the request
+   *
+   * @return {Object} Payment request
+   */
+  function createRequest (params) {
+    const paymentRequest = {
+      address: params.destination_account,
+      amount: params.destination_amount,
+      expires_at: params.expires_at || moment().add(defaultRequestTimeout, 'seconds').toISOString()
+    }
+
+    if (params.data) {
+      paymentRequest.data = params.data
+    }
+
+    paymentRequest.address += '.' + (params.id || uuid.v4())
+
+    const sharedSecret = Buffer.from(params.shared_secret, 'base64')
+    const conditionPreimage = hmacHelper.hmacJsonForSspCondition(paymentRequest, sharedSecret)
+    const condition = toConditionUri(conditionPreimage)
```

Comment:
They're the same thing! ^^

`cc` == `util/condition`. The only reason it's called `cc` in the receiver is because it uses both `toConditionUri` and `toFulfillmentUri` from `util/condition`. :)